content/docs/building-apps/first-deposit.mdx

```diff
@@ -0,0 +1,58 @@
+
+### Option one: the anchor creates and funds the Stellar account
+
+For this option, the wallet needs to allow users to initiate their first deposit without having to add an asset/establish a trustline. The wallet then prompts the user to add the trustline once funds are received by the anchor. The flow looks like this: 
+
+1. The wallet registers a new user and issues a keypair 
+2. The wallet initiates the first deposit on behalf of the user without requiring the user to add the asset/create the trustline
+3. The anchor provides deposit instructions to the customer 
+4. The user transfers money from a bank account to the anchor’s bank account
+5. Once the anchor receives the transfer, the anchor creates and funds the Stellar account for the customer. *Note*: An anchor should always maintain a healthy amount of XLM in their distribution account to support new account creations. If doing so becomes unsustainable, it’s recommended that the anchor collaborates with wallets to determine a strategy based on the number of account creation requests. The recommended amount is 2XLM per user account creation (1XLM to meet the minimum balance requirement, and 1XLM for establishing trustlines and covering transaction fees).
+6. The wallet detects that the account has been created and a trustline must be established.
+7. The wallet prompts the user to add the asset/create the trustline. 
+8. Finally, the anchor sends the deposit funds to the user’s Stellar account
+
+![anchor creates account flow](../../web/assets/first-deposit-anchor-flow.png)
+
+With the flow we just described above, the wallet and the anchor have to facilitate listening for and responding to the trustline status, which can create user experience frictions when waiting for the trustline to be established. To address this issue, Protocol 15 introduceClaimable Balances, which enhance the flow by allowing users to start using the wallet without having to secureXLM, or to wait to create the trustline after they made the first deposit. Both the wallet and the anchor have to implement Claimable Balance support in order to make this flow work. 
+
+The flow with Claimable Balances looks like this, 
+
+1. The wallet registers a new user, and generates a keypair 
+2. The wallet initiates a deposit on behalf of a user
+3. The anchor provides deposit instructions to the wallet 
+4. The user transfers money from a bank account to the anchor’s account
+5. The anchor creates and funds the user's Stellar account plus the amount required for trustlines and transaction fees.  Again, we suggest 2 XLM to start. 
+6. The anchor creates a Claimable Balance 
+7. The wallet detects the Claimable Balance for the account, claims the funds, and posts it in the wallet. 
+
+![anchor creates account claimable balance flow](../../web/assets/first-deposit-claimable-balance-flow.png)
+
+### Option two: The wallet creates and funds the Stellar account upon user sign-up. 
```

Comment:
```suggestion
### Option 2: The wallet creates and funds the Stellar account upon user sign-up
```
Again, I find numbers clearer. Also, we either want a period here _and_ in Option 1, or we don't want a period here either.